libvips/foreign/pngload.c

```diff
@@ -395,6 +407,10 @@ vips_foreign_load_png_buffer_init( VipsForeignLoadPngBuffer *buffer )
  * @out: (out): decompressed image
  * @...: %NULL-terminated list of optional named arguments
  *
+ * Optional arguments:
+ *
+ * * @unlimited: %gboolean, allow any number of text chunks
+ *
  * Read a PNG file into a VIPS image. It can read all png images, including 8-
  * and 16-bit images, 1 and 3 channel, with and without an alpha channel.
  *
```

Comment:
Add a para, eg. "By default, the PNG loader limits the number of text and data chunks to block some denial of service attacks. Set @unlimited to disable these limits."